app/helpers/notifications_helper.rb

```diff
@@ -0,0 +1,8 @@
+module NotificationsHelper
+  def create_system_notification(title, message)
+    users = User.where.not(confirmed_at: nil)
+    users.each do |u|
```

Comment:
When iterating over large collections, it is **very important** to use `.find_each` instead of `each` (`each` loads all queried objects in memory, which could be deadly when iterating over whole tables, while `find_each` loads objects in batches).